webservice/src/main/resources/public/assets/401.html

```diff
@@ -27,7 +27,7 @@
     width: 1140px;
     margin: 0 auto;
     padding: 100px 0 50px;
-    background-image: url('back-default.png');
+    background-image: url('ui/back-default.png');
```

Comment:
@shahsank3t Can you please check whether this change is right as I am not sure about html